wsd/DocumentBroker.cpp

```diff
@@ -2148,8 +2147,11 @@ std::size_t DocumentBroker::removeSession(const std::string& id)
                 LOG_DBG("No more sessions after removing last. Setting MarkToDestroy flag.");
             }
         }
-        else
-            assert(!_docState.isMarkedToDestroy());
+        else if (activeSessionCount > 0)
+        {
+            assert(!_docState.isMarkedToDestroy() &&
```

Comment:
Would be good to ERR this into the log too, even in the non-debug case (ie. asserts compiled out), so that we have a sign of something very bad happening.